support-frontend/assets/components/paymentButton/defaultPaymentButtonContainer.tsx

```diff
@@ -61,7 +61,7 @@ export function DefaultPaymentButtonContainer({
 		(state) => state.common.internationalisation,
 	);
 
-	const inThreeTierVariant = inThreeTierV2Variant(
+	const inThreeTier = showThreeTierCheckout(
```

Comment:
`inThreeTier` is only used in 1 place, maybe we can remove the variable and just call `showThreeTierCheckout` in the 1 place it's used? It's a small simplification, but means less code / less variable names to consider.